src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/DJApplicantReceivedNotificationHandler.java

```diff
@@ -68,10 +71,16 @@ private String identifyTemplate(CaseData caseData) {
             template = notificationsProperties.getApplicantSolicitor1DefaultJudgmentRequested();
             templateReference = REFERENCE_TEMPLATE_REQUESTED;
         }
-        if (ofNullable(caseData.getRespondent2()).isEmpty()) {
+        if (ofNullable(caseData.getRespondent2()).isEmpty() && !caseData.isLipvLipOneVOne()) {
             template = notificationsProperties.getApplicantSolicitor1DefaultJudgmentReceived();
             templateReference = REFERENCE_TEMPLATE_RECEIVED;
         }
+        if (ofNullable(caseData.getRespondent2()).isEmpty()
```

Comment:
no need of 1st check

Reply:
1st check was already there, it's for more than 1 respondent.

Reply:
Review it and you was correct, I've fix it.